src/pageTypes/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { sectionNames } from "@saleor/intl";
+import { asSortParams } from "@saleor/utils/sort";
+import { parse as parseQs } from "qs";
+import React from "react";
+import { useIntl } from "react-intl";
+import { Route, RouteComponentProps, Switch } from "react-router-dom";
+
+import { WindowTitle } from "../components/WindowTitle";
+import {
+  pageTypeAddPath,
+  pageTypeListPath,
+  PageTypeListUrlQueryParams,
+  PageTypeListUrlSortField,
+  pageTypePath,
+  PageTypeUrlQueryParams
+} from "./urls";
+import PageTypeCreate from "./views/PageTypeCreate";
+import PageTypeListComponent from "./views/PageTypeList";
+import PageTypeUpdateComponent from "./views/PageTypeUpdate";
```

Comment:
Just to match convention

```suggestion
import PageTypeDetailsComponent from "./views/PageTypeDetails";
```